Schemas/visinfo.xsd

```diff
@@ -139,6 +140,25 @@
         <xs:attribute name="SpaceBoundariesVisible" type="xs:boolean" default="false"/>
         <xs:attribute name="OpeningsVisible" type="xs:boolean" default="false"/>
     </xs:complexType>
+    <xs:complexType name="ComponentTranslucency">
+        <xs:sequence>
+            <xs:element name="TranslucencySetupHints" type="TranslucencySetupHints" minOccurs="0"/>
+            <xs:element name="Exceptions" minOccurs="0">
+                <!-- Inverses DefaultTranslucency and TranslucencySetupHints  -->
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="Component" type="Component" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
From @pjanck: Use `minOccurs="1"` in the inner list, if the Exception object is present, then it should have at least one exception.